clients/src/test/java/org/apache/kafka/clients/producer/internals/BuiltInPartitionerTest.java

```diff
@@ -77,7 +86,7 @@ public void testStickyPartitioning() {
         assertNotEquals(partA, builtInPartitionerA.peekCurrentPartitionInfo(testCluster).partition());
 
         // Check that switching works even when there is one partition.
-        BuiltInPartitioner builtInPartitionerB = new SequentialPartitioner(logContext, TOPIC_B, 1);
+        BuiltInPartitioner builtInPartitionerB = new SequentialPartitioner(logContext, TOPIC_B, 1, false, "");
```

Comment:
```suggestion
        BuiltInPartitioner builtInPartitionerB = new SequentialPartitioner(logContext, TOPIC_B, 1, rackAware, clientRackId);
```

Just to make the meaning of `false` and `""` obvious at a glance :)